api/src/main/java/com/netflix/iceberg/Table.java

```diff
@@ -171,4 +172,10 @@ default AppendFiles newFastAppend() {
    * @return a new {@link Transaction}
    */
   Transaction newTransaction();
+
+  /**
+   * @return a {@link FileIO} to read and write table data and metadata files
+   */
+  FileIO io();
```

Comment:
Should `TableOperations#io` continue to exist?

Reply:
Yes, because `TableOperations` is still how implementations are passed.